interfaces/builtin/libvirt.go

```diff
@@ -35,6 +36,8 @@ sendto
 sendmsg
 socket
 socketpair
+listen
+accept
```

Comment:
This is curious. What in the client (ie, the plugging side) is setting up a listener?